### PR TITLE
BZ1827550-update-custom-node-selector

### PR DESCRIPTION
**Pull request by lamek**
@timflannagan1 per your PR for https://bugzilla.redhat.com/show_bug.cgi?id=1827550

https://github.com/openshift/openshift-docs/pull/21486/files

### CODE REVIEW

**Review by timflannagan — APPROVED** [review]
Thanks, this seems much clearer to me.

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 1]
```diff
@@ -82,22 +82,24 @@ spec:
...
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": "" <1>
 ----
+<1> Add a `nodeSelector` parameter with the appropriate value to the component you want to move. You can use a `nodeSelector` in the format shown or use `<key>: <value>` pairs, based on the value specified for the node.
```
Comment (bobfuru) [inline metering/configuring_metering/metering-common-config-options.adoc:104]: s/<key>: <value>` pairs,/<key>:<value>` pairs,/
or just `key-value pairs`, which is generally how I've written it in previous orgs.

**Review by bobfuru — COMMENTED** [review]
One little nit and LGTM

**Comment by timflannagan1** [comment]
@lamek can you retitle this PR to have the `Bug 1827550: ...` prefix so it gets picked up by the bugzilla bot?

**Pull request closed without merging**